doc/specs/#1595 - Suggestions UI/Snippets.md

```diff
@@ -490,14 +564,31 @@ various tools. Stored publicly on a GitHub repo (a la the winget-pkgs repo).
 Users can submit Snippets with descriptions of what the Snippet does. The
 Terminal can plug into that repo automatically and fetch the latest community
 commands, immediately giving the user access to a wide berth of common
-Snippets. That could easily be done as another suggestion source (in the same
-vein as `local` is.)
+Snippets. That could easily be done as another suggestion source.
+
+#### Profiles in `.wt.json`
+
+If we've got a `.wt.json` in a given directory, should we be dynamically
+adding/removing other settings too? Wouldn't profiles also make sense? Take for
+example, the Terminal repo. We've got a PowerShell build environment and a CMD
+one. What if we could drop two profiles in the `.wt.json` file, with the
+`commandline`'s set up to call those scripts as needed?

```

Comment:
IIRC there's also been discussion before about snippets that only work in some profiles. It'd be cool if I had a snippet for "Build Project" that had CMD syntax vs PowerShell syntax.

Idk exactly _how_ to solve that problem, but yeah, definitely something to consider

Reply:
This might actually be a really important thing too. I'm thinking about some nuances between CMD, PowerShell, and Linux. `git` commands are generally easy to copy/paste between the two, but the second you start talking about escaping characters and stuff like that, this can be annoying and difficult.